README.md

```diff
@@ -79,6 +79,19 @@ All of the arguments are optional:
 
 `--parsers`, `--detectors` and `--specials`: These arguments are for advanced usage. They provide an easy way to customize the file parser and dependency detection. Check [the pluggable design document](https://github.com/depcheck/depcheck/blob/master/doc/pluggable-design.md) for more information.
 
+## Usage with a configuration file
+
+Depcheck can be used along with rc configuration files. In order to do so, create a .depcheckrc file in your project's package.json folder, and set the CLI keys in YAML format.
+For example, the CLI arguments `--ignores="eslint,babel-*" --skip-missing=true` would turn into:
```

Comment:
worth mentioning here what formats are supported rather than just YAML (as i think you said it can do JSON too?)

Reply:
Good catch, updated